helm-chart/values.schema.json

```diff
@@ -281,20 +330,52 @@
                   "type": "string"
                 }
               },
-              "required": ["namespace", "issuerName"]
+              "required": [
+                "namespace",
+                "issuerName"
+              ]
             }
           },
           "oneOf": [
-            {"$ref": "#/definitions/emptyObject"},
-            {"required": ["disk"]},
-            {"required": ["awsPCA"]},
-            {"required": ["awsSecret"]},
-            {"required": ["vault"]},
-            {"required": ["certManager"]}
+            {
+              "$ref": "#/definitions/emptyObject"
+            },
+            {
+              "required": [
+                "disk"
+              ]
+            },
+            {
+              "required": [
+                "awsPCA"
+              ]
+            },
+            {
+              "required": [
+                "awsSecret"
+              ]
+            },
+            {
+              "required": [
+                "vault"
+              ]
+            },
+            {
+              "required": [
+                "certManager"
+              ]
+            }
```

Comment:
Is this necessary? It's much easier to read these when they're all flattened. I think it's fine to add newlines to the enums and basic string lists, but changing these objects feels unnecessary.

Reply:
No, these aren't necessary. Just something the formatter did. I'll revert them.

Reply:
@sjberman, flattened those objects and 0-2 length lists. Left the newlines for lists 3+ long. Found and removed a duplicate telemetry object in the process.